tests/test-awful-tag.lua

```diff
@@ -128,4 +128,117 @@ end
 end
 }
 
+local multi_screen_steps = {}
+
+-- Clear the state (this step will be called once per disposition)
+table.insert(multi_screen_steps, function()
+    if #client.get() == 0 then
+        -- Remove all existing tags created by the rc.lua callback
+        for s in screen do
+            while #s.tags > 0 do
+                local t = s.tags[1]
+                -- Test #1051 against regressions
+                assert(t.index == 1)
+
+                assert(t.screen == s)
+
+                t:delete()
+                assert(not t.activated)
+                assert(t.screen == nil)
+            end
+        end
+
+        return true
+    end
+
+    for _, c in ipairs(client.get()) do
+        c:kill()
+    end
```

Comment:
Isn't the killing already being done by `_multi_screen` and thus unnecessary here?